plugins/jira/skills/status-analysis/SKILL.md

```
@@ -362,6 +363,15 @@ h2. Status Rollup From: {start-date} to {end-date}
      ** Risk 1 (or "None at this time")
 ```
 
+**For `feature_markdown` (generate-feature-updates)**:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)